modin/experimental/engines/pandas_on_ray/io_exp.py

```diff
@@ -172,6 +181,43 @@ def read_sql(
             cls.frame_cls(np.array(partition_ids), new_index, cols_names)
         )
 
+    @classmethod
+    def to_pickle(cls, qc, **kwargs):
+        """
+        In experimental mode, all partitions are written to their own separate file.
```

Comment:
Is this useful? How would someone reuse this group of pickle files somewhere else?